Regula_Falsi_Method/regula_falsi.cpp

```diff
@@ -0,0 +1,71 @@
+/* Girlscript Summer Of Code
+Regula Falsi Method in c
```

Comment:
this should be `Regula Falsi Method in c`

Reply:
U mean C++ right?